### PR TITLE
[Localization] #1761 Korean trainer dialogue partial update

### PR DESCRIPTION
**Pull request by returntoice**
## What are the changes?

Translating locales\ko\dialogue.ts into Korean.

@sodaMelon @Ohmry @returntoice now participating in #1761 translating project.

## Why am I doing these changes?

Only rival's dialogue has been translated into Korean yet.

## What did change?

I've just translated double battles' dialogues.

다른 대사들은 얼추 원문에 맞게 번역하였으나, 풍&란의 대사는 원문과 조금 달라서 이상하다고 생각하실 수도 있습니다. 오루알사의 대사를 그대로 가져오다 보니 원본과는 내용이 조금 달라졌습니다.

## How to test the changes?

It seems there won't be much to test since it's just a translation of an existing .ts file. You can check it when you meet the respective trainers.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by sodaMelon** [comment]
> 
> 다른 대사들은 얼추 원문에 맞게 번역하였으나, 풍&란의 대사는 원문과 조금 달라서 이상하다고 생각하실 수도 있습니다. 오루알사의 대사를 그대로 가져오다 보니 원본과는 내용이 조금 달라졌습니다.


"일부분은 한국에 정식발매한 포켓몬 게임 대사에 맞췄습니다. 포켓로그의 원문과 내용이 조금 다를 수 있습니다." <-를 이슈 사항에 추가하겠습니다.

**Comment by sodaMelon** [comment]
anyone who can add Reviewers, add me to this pull request. 

hi @Madmadness65  Can you do this?

File: src/locales/ko/dialogue.ts [hunk 1]
```diff
@@ -2335,113 +2335,113 @@ export const PGFmiscDialogue: SimpleTranslationEntries = PGMmiscDialogue;
...
   },
   "tate_liza_double": {
     "encounter": {
-      1: `Tate: Are you suprised?
-                  $Liza: We are two gym leaders at once!
-                  $Tate: We are twins!
-                  $Liza: We dont need to talk to understand each other!
-                  $Tate: Twice the power...
-                  $Liza: Can you handle it?`,
+      1: `풍: 에헤헤... 체육관 관장이
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:2360]: 이 부분이 원본과 다른 부분이군요? 이정도면 의미가 퇴색되진 않은 것 같습니다.

File: src/locales/ko/dialogue.ts [hunk 2]
```diff
@@ -2335,113 +2335,113 @@ export const PGFmiscDialogue: SimpleTranslationEntries = PGMmiscDialogue;
...
+                  $란: 두 명이나 있어서 놀랐지?
+                  $풍: 우리는 쌍둥이!
+                  $란: 굳이 말을 하지 않아도 서로가 무슨 생각을 하고 있는지
+                  $풍: 자동으로 머릿속에 떠오르니까
+                  $란: 호흡을 척척 맞출 수가 있지!`,
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:2365]: 아마 일본어<->영어로 오게되면서, 내용 구성이 바뀐것 같습니다. 이정도면 괜찮은 것 같아요

**Review by sodaMelon — APPROVED** [review]
hi. i checked them than you metioned about.
there is a no problem to understand.  
i approved this pull request.

**Comment by Tempo-anon** [comment]
Is this good to be merged in or do you want to keep working on this PR? Also, the double battles have been removed from the game (for now)

**Comment by sodaMelon** [comment]
> Is this good to be merged in or do you want to keep working on this PR? Also, the double battles have been removed from the game (for now)

oh... I had fun playing the double battle, but it's a shame.
Please merge it for now. (You might use it later.)

**Review by Tempo-anon — APPROVED** [review]
Looks good but there's a lot of diff changes since it seems like a space was added after every gym leader encounter

**Comment by returntoice** [comment]
> Looks good but there's a lot of diff changes since it seems like a space was added after every gym leader encounter

I just copied them all from dialogue.ts in EN locales folder. The current KO dialogue.ts doesn't have a space after every encounter part, but EN dialogue.ts has. can it cause problems?

**Pull request merged**